app/controllers/products_controller.rb

```diff
@@ -60,12 +60,15 @@ def update
   def destroy
     respond_to do |format|
       if @product.destroy
-        format.html {redirect_to products_url, notice: t("products.destroyed")}
+        format.html {redirect_to products_url, notice: t("products.destroy.success")}
```

Comment:
```suggestion
        format.html {redirect_to products_url, notice: t(".success")}
```
Nếu đặt đúng format rồi thì có thể call rút gọn e nhé
Fix tương tự những chỗ khác